swarm/src/behaviour.rs

```diff
@@ -69,16 +69,20 @@ pub trait NetworkBehaviour: Send + 'static {
 
     /// Creates a new `ProtocolsHandler` for a connection with a peer.
     ///
-    /// Every time an incoming connection is opened, and every time we start dialing a node, this
-    /// method is called.
+    /// Every time an incoming connection is opened, and every time another [`NetworkBehaviour`]
+    /// emitted a dial request, this method is called.
     ///
     /// The returned object is a handler for that specific connection, and will be moved to a
     /// background task dedicated to that connection.
     ///
-    /// The network behaviour (ie. the implementation of this trait) and the handlers it has
-    /// spawned (ie. the objects returned by `new_handler`) can communicate by passing messages.
-    /// Messages sent from the handler to the behaviour are injected with `inject_event`, and
-    /// the behaviour can send a message to the handler by making `poll` return `SendEvent`.
+    /// The network behaviour (ie. the implementation of this trait) and the handlers it has spawned
+    /// (ie. the objects returned by `new_handler`) can communicate by passing messages. Messages
+    /// sent from the handler to the behaviour are injected with [`NetworkBehaviour::inject_event`],
+    /// and the behaviour can send a message to the handler by making [`NetworkBehaviour::poll`]
+    /// return [`NetworkBehaviourAction::SendEvent`].
```

Comment:
Did you rename it? Otherwise I think this is actually called `NotifyHandler` :)

Reply:
:+1: It was previously `SendEvent`, I changed it to `['NetworkBehaviour::SendEvent']`. https://github.com/libp2p/rust-libp2p/pull/2191/commits/4d0faf9af81a5b1f3373b062e2a7cb498eb97dba changes it to `['NetworkBehaviour::NotifyHandler']`.